core/src/main/resources/jenkins/security/whitelisted-classes.txt

```diff
@@ -135,34 +135,10 @@ org.apache.commons.fileupload.disk.DiskFileItem
 org.apache.commons.fileupload.util.FileItemHeadersImpl
 org.apache.tools.ant.Location
 
-# TODO see main ruby-runtime section below
-org.jenkinsci.jruby.JRubyMapper$DynamicProxy
-
 # TODO remove when https://github.com/jenkinsci/xtrigger-lib/pull/9 is widely adopted in fstrigger-plugin, urltrigger-plugin, etc.
 org.jenkinsci.lib.xtrigger.XTriggerCause
 org.jenkinsci.lib.xtrigger.XTriggerCauseAction
 
-# TODO remove (also XStream2.BlacklistedTypesConverter.JRUBY_PROXY) when Ruby Runtime is fixed
-# Related PRs:
-#   - https://github.com/jenkinsci/ruby-runtime-plugin/pull/5,
```

Comment:
I've archived this repo and jenkins.rb 😄

Reply:
As of jenkins-infra/update-center2#565 these need to be archived as well:

- https://github.com/jenkinsci/buddycloud-plugin
- https://github.com/jenkinsci/capitomcat-plugin
- https://github.com/jenkinsci/chef-plugin
- https://github.com/jenkinsci/ci-skip-plugin
- https://github.com/jenkinsci/commit-message-trigger-plugin
- https://github.com/jenkinsci/gitlab-hook-plugin
- https://github.com/jenkinsci/git-notes-plugin
- https://github.com/jenkinsci/installshield-plugin
- https://github.com/jenkinsci/jenkins.py
- https://github.com/jenkinsci/pathignore-plugin
- https://github.com/jenkinsci/pyenv-plugin
- https://github.com/jenkinsci/rbenv-plugin
- https://github.com/jenkinsci/rvm-plugin
- https://github.com/jenkinsci/singleuseslave-plugin
- https://github.com/jenkinsci/travis-yml-plugin
- https://github.com/jenkinsci/yammer-plugin

Reply:
done thanks for the list